sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ServiceBusProcessor.cs

```diff
@@ -1035,6 +1058,24 @@ public async ValueTask DisposeAsync()
             GC.SuppressFinalize(this);
         }
 
+        /// <summary>
+        ///
+        /// </summary>
+        /// <param name="maxConcurrentCalls"></param>
+        /// <param name="prefetchCount"></param>
+        public void UpdateConcurrency(int maxConcurrentCalls, int prefetchCount)
+        {
+            Argument.AssertAtLeast(maxConcurrentCalls, 1, nameof(maxConcurrentCalls));
+            Argument.AssertAtLeast(prefetchCount, 0, nameof(prefetchCount));
+            lock (_maxConcurrencySyncLock)
```

Comment:
The sync lock name is probably now also no longer good and needs a more descriptive name